infrastructure/lambda/template.yaml

```diff
@@ -476,6 +492,9 @@ Resources:
     Type: AWS::Serverless::Function
     Properties:
       Handler: uk.gov.di.ipv.cri.address.api.handler.IssueCredentialHandler::handleRequest
+      DeploymentPreference:
+        Alarms:
+          - !Ref IssueCredentialFunctionCanaryErrors
```

Comment:
Issue credential sits behind the public endpoint `/credential/issue` are we able to also have an alarm for that please?

Reply:
thank you for that, added 5xx error alarm for issue credential